climada/util/api_client.py

```diff
@@ -335,12 +345,12 @@ def get_data_types(self, data_type_group=None):
         -------
         list of DataTypeInfo
         """
-        url = f'{self.host}/rest/data_types'
+        url = f'{self.url}/data_type'
         params = {'data_type_group': data_type_group} \
             if data_type_group else {}
         return [DataTypeInfo(**jobj) for jobj in Client._request_200(url, params=params)]
 
-    def get_data_type(self, data_type):
+    def get_data_type_info(self, data_type):
         """Returns the data type from the climada data API
```

Comment:
That seems contractictory. The `data_type` is specified as an input parameter, so the method cannot return the data type.

Reply:
You got a point there. What would you suggest? data_type_name? If we change it we should change it all over the place, including the `DataTypeInfo` class.